### DISCUSSION TITLE
Possibility of using modules instead of separated resources

### DISCUSSION THREAD

**Discussion by tdiazvelez** [GoogleCloudPlatform/terraformer#1454] [General]
Hey everyone, it could be possible to have the option to do it? 

For example, using the **module** [aws_vpc](https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/latest) including subnet, AZ's, etc configurations, instead of using 2 (or more) separated **resources** like, [aws_subnet](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/subnet) and [aws_vpc](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc)

By other way, its kinda hard working with the files when you import them from AWS and you want to unify, for example some different regions into one file, because for everything, you have different folders, with own variables, outputs, and very long .tfstate's

Anyway, thanks for the amazing job! Keep pushing!

**Comment by XerXi** [comment]
Hi,
For the many folders and files problem - have you tried using the arguments "--compact --path-pattern ." ?

**Reply by tdiazvelez** [reply]
Hi, do you have an example of usage??  I was using it this way 

`terraformer import aws --regions=eu-west-1,eu-west-3,us-east2 --resources=vpc,subnet,s3,ec2_instance,sg --connect=true`

**Reply by XerXi** [reply]
ye just add at the end
terraformer import aws --regions=eu-west-1,eu-west-3,us-east2 --resources=vpc,subnet,s3,ec2_instance,sg --connect=true --compact --path-pattern .
